package.json

```diff
@@ -99,26 +97,34 @@
     "amphtml-validator": "^1.0.23",
     "babel-core": "^6.26.3",
     "babel-eslint": "^10.0.0",
-    "babel-loader": "^8.0.0",
+    "babel-loader": "^7.1.5",
```

Comment:
Downgraded as Babel Loader v8 requires Babel plugins to adopt the v7 API, and I don't want to upgrade to Babel 7 in this PR (we'll do that later).